types/uint8arrays/index.d.ts

```diff
@@ -3,3 +3,11 @@ declare module 'uint8arrays' {
     export function fromString(s: string, enc?: string): Uint8Array
     export function concat(bs: Array<Uint8Array>): Uint8Array
 }
+
+declare module 'uint8arrays/concat' {
+    export default function concat(bs: Array<Uint8Array>): Uint8Array
+}
+
+declare module 'uint8arrays/to-string' {
+    export default function toString(b: Uint8Array, enc?: string): string
+}
```

Comment:
Doesn't v 2.0.x include types?

Reply:
It does, sort of. https://github.com/achingbrain/uint8arrays/issues/13

Reply:
grr, ok thanks.